tools/generator/test/GeneratorTests.swift

```diff
@@ -17,7 +17,8 @@ final class GeneratorTests: XCTestCase {
             targets: Fixtures.targets,
             targetMerges: [:],
             invalidTargetMerges: ["Y": ["Z"]],
-            extraFiles: []
+            extraFiles: [],
+            bazelWorkspaceName: "bazel_workspace"
```

Comment:
Can this be moved right after `name`, towards the top?

Reply:
Done